tests/features/data-visibility-and-review.feature

```diff
@@ -1,122 +1,202 @@
+    Given a public user requests a CSV export
+    And the dataset contains public and internal records with different review_status values
+    When the download is prepared
+    Then only the public records should be included
+    And the download should list the review_status for each record
+
+  # ---------------------------------------------------------------------------
+  # AUTHENTICATED STAFF ACCESS
+  # ---------------------------------------------------------------------------
+
+  @staff_access @visibility
+  Scenario Outline: Staff can access all data and its review status
+    Given I am an authenticated staff member
+    When I view <data_type> data through internal endpoints
+    Then I should see visibility internal and public records
+    And I should see whether each record is provisional or approved
+
+    Examples:
+      | data_type    |
+      | things       |
+      | locations    |
+      | samples      |
+      | observations |
+
+  @staff_access @management
+  Scenario: Staff can change visibility without affecting review status
+    Given data is stored with visibility internal and review_status provisional
+    When an authenticated staff member updates the visibility to public
+    Then the data visibility should be persisted as public
+    And the review_status should remain provisional
+    And retrieving the data should show the updated visibility and original review_status
+
+  @staff_access @review_status
+  Scenario: Staff can approve data without changing visibility
+    Given data is stored with visibility public and review_status provisional
+    When an authenticated staff member updates the review_status to approved
+    Then the data should remain visible to the public
+    And the review_status should be persisted as approved
+    And the API response should immediately show the approved status
+
+  # ---------------------------------------------------------------------------
+  # DATA SUBMISSION AND WORKFLOW
+  # ---------------------------------------------------------------------------
+
+  @workflow @validation
+  Scenario Outline: New records require explicit visibility and review status
+    When I attempt to create a new <data_type> without both visibility and review_status
+    Then the request should be rejected with a 422 status code
+    And the error response should indicate that visibility is required
+    And the error response should indicate that review_status is required
+
+    Examples:
+      | data_type    |
+      | thing        |
+      | contact     |
+      | observation |
+
+  @workflow @defaults
+  Scenario: Safe defaults when both attributes are provided
+    Given data is being submitted to the system with visibility internal and review_status provisional
+    When the record is created
+    Then the system should persist those explicit values as provided
+
+  @workflow @privacy_protection
+  Scenario: Private contact data is protected by default
```

Comment:
If _all_ contact data is private by default,  I think that this scenario should be "Contact data is private and protected by default," and then the rest of the scenario describes the workflow of setting the default values.